src/aat/java/uk/gov/hmcts/ccd/datastore/befta/RoleAssignmentSetup/RoleAssignmentSetup.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.ccd.datastore.befta.RoleAssignmentSetup;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.specification.RequestSpecification;
+import uk.gov.hmcts.ccd.datastore.tests.AATHelper;
+import uk.gov.hmcts.ccd.datastore.tests.helper.idam.AuthenticatedUser;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class RoleAssignmentSetup{
+
+    public static final String CCD_CASEWORKER_AUTOTEST_PASSWORD = "CCD_CASEWORKER_AUTOTEST_PASSWORD";
+    public static final String SOLICITOR_1_ORG_ROLE_ASSIGNMENTS = "/Users/ashleynoronha/code/ccd/ccd-data-store-api/src/aat/java/uk/gov/hmcts/ccd/datastore/befta/RoleAssignmentSetup/OrganisationRoleAssignmentsSolicitor1.json";
+    public static final String SUPERUSER_ORG_ROLE_ASSIGNMENTS = "/Users/ashleynoronha/code/ccd/ccd-data-store-api/src/aat/java/uk/gov/hmcts/ccd/datastore/befta/RoleAssignmentSetup/OrganisationRoleAssignmentsSuperUser.json";
+
+    public static void main(String[] args) {
+        new RoleAssignmentSetup().setupOrganisationRoleAssignments();
+    }
+
+//    TODO CHANGE THE PATH OF THE 2 JSON FILES IN THE STATIC VARS ABOVE TO THE CORRECT PATH FOR YOUR LOCAL MACHINE
+//    TODO REPLACE THE IDAM ID IN THE 2 JSON FILES LISTED ABOVE AS FINAL VARS WITH CORRECT IDAM IDS FROM YOUR LOCAL
+
+        //NO LONGER REQUIRED AS WE CAN JUST POST ASSIGNMENTS WITH THE 'REPLACE EXISTING' FLAG
+    //    public ArrayList<RoleAssignment> getRoleAssignements(){
+    //         ArrayList<LinkedHashMap> roleAssingments = asRoleAssignemntCaseworker()
+    //            .get("/am/role-assignments/actors/82853c08-73dd-4bd3-a50e-7dae355da504")
+    //            .then()Befta_Default_Token_Creation_Data_For_Case_Creation
+    //            .extract().path("roleAssignmentResponse");
+    //
+    //         ArrayList<RoleAssignment> raList = (ArrayList<RoleAssignment>) roleAssingments.stream().map(ra ->
+    //             RoleAssignment.builder()
+    //             .roleName((String) ra.get("roleName"))
+    //             .roleType((String) ra.get("roleType"))
+    //             .grantType((String) ra.get("grantType"))
```

Comment:
delete these commented lines if needed we can get from history

Reply:
DONE